### DISCUSSION TITLE
Top Clients is showing IPv6 addresses as [IPVAddress]%eth0

### DISCUSSION THREAD

**Discussion by RoyalClem** [AdguardTeam/AdGuardHome#6622] [Q&A] [answered]
For some reason, since pointing AdGuard Home at all interfaces, instead of just eth0 it is showing Top Clients incorrectly.
![image](https://github.com/AdguardTeam/AdGuardHome/assets/24209472/d7ef9d5a-4852-426d-8b36-af4a34576827)

When I look at the client, for example the first client, it shows no query's found...
![image](https://github.com/AdguardTeam/AdGuardHome/assets/24209472/f9fc22b2-854a-47af-89f9-348e247634ab)

However, once I remove %eth0, it works fine.
![image](https://github.com/AdguardTeam/AdGuardHome/assets/24209472/9fa85cb2-2489-42ac-9bc3-cf947193e066)

Did I set up something wrong? Should I try a different version of AdGuard Home? Currently running Version: 
v0.107.43.

**Comment by RoyalClem** [comment]
For bonus points, is there a way i can get the client list to show the names of the device? Currently, IPv6 is not showing the correct names, just showing the ipv6 address.

**Comment by lespea** [comment, marked as answer]
Those are link local ipv6 addresses which need to specify the interface they're for.  [Random stack overflow post on the matter](https://stackoverflow.com/a/45796496).

**Reply by RoyalClem** [reply]
THANK YOU! Which IPv6 address should I use then? I am guessing the fdf1 address because if I use the 2601 the client list just points to the WHOIS.

![Screenshot 2024-01-07 123220](https://github.com/AdguardTeam/AdGuardHome/assets/24209472/c68ad604-2847-4390-8446-b32e350e0f28)


Also, I may need your help (if possible) to get my reverse DNS resolvers working right.
